README.md

```diff
@@ -977,3 +980,8 @@ To change it:
 1. Go to `Atom -> Preferences -> Packages`.
 2. Search for the "language-julia" package and open the settings for it.
 3. Find preferred line length (under "Julia Grammar") and change it to 92.
+
+## Copyright waiver
+
+This work is in the public domain.
+Copyright and related rights are waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Seems redundant now. Should we just have the LICENSE file?

Reply:
yeah, that seems fair enough 

I liked linking to the "human-readable" version, but we just use a LICENSE file everywhere else :) Will change

Reply:
done